.github/workflows/ci.yaml

```diff
@@ -166,3 +166,17 @@ jobs:
       - name: Test SGX & FPGA Admission Webhook, Deploy Operator
         run: |
           make test-with-kind
+
+  terrascan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Install terrascan
```

Comment:
There's a github action for this too but below is fine as it can be copied to `Jenkinsfile` too.

Reply:
I've looked at it and found it hard to use in our setup due to a requirement to use iac_dir input to point out to the directory with kustomization.yaml. This would require to create one step per directory in the ci.yaml and maintain that.

Reply:
> This would require to create one step per directory in the ci.yaml and maintain that.

OK, I see. So the matrix syntax cannot be used?

Reply:
I'd not say it can't be used, but it would be much less straightforward and reusable (locally and in Jenkins) and much more hacky. The main challenge with it is a difficulty to generate matrix targets. Official [github documentation](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs) only shows those as a list of constants.

Reply:
+1